pyDFT/mac_function.py

```diff
@@ -0,0 +1,19 @@
+def roman_numeral(s: str) -> int:
+    """Takes a roman numeral, entered as 's' a string, and returns the corresponding
+    value as an integer
+
+
+    """
+    if s == "I":
+        return 1
+    if s == "IV":
+        return 4
+    if s == "V":
+        return 5
+    if s == "X":
+        return 10
+
+    if not isinstance(s, str):
+        raise ValueError(f"x must be a string; {s} is invalid")
+    while s != "I" and s != "IV" and s != "V" and s != "X":
```

Comment:
This is the right idea but you want an if-statement here instead of a while. Also it would be more efficient to do `if not s in {"I", "IV", "V", "X"}` than the if statement here